src/views/catalog/CreateFromInstanceTypes/components/VMDetailsSection/components/DynamicSSHKeyInjectionIntanceType.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { useInstanceTypeVMStore } from '@catalog/CreateFromInstanceTypes/state/useInstanceTypeVMStore';
+import { DynamicSSHKeyInjection } from '@kubevirt-utils/components/DynamicSSHKeyInjection/DynamicSSHKeyInjection';
+import HelpTextIcon from '@kubevirt-utils/components/HelpTextIcon/HelpTextIcon';
+import VirtualMachineDescriptionItem from '@kubevirt-utils/components/VirtualMachineDescriptionItem/VirtualMachineDescriptionItem';
+import { t } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { PopoverPosition } from '@patternfly/react-core';
+
+const DynamicSSHKeyInjectionIntanceType = () => {
+  const { instanceTypeVMState, setInstanceTypeVMState } = useInstanceTypeVMStore();
+  const { sshSecretCredentials } = instanceTypeVMState;
+
+  return (
+    <VirtualMachineDescriptionItem
+      descriptionData={
+        <DynamicSSHKeyInjection
+          onSubmit={(checked: boolean) => {
+            setInstanceTypeVMState({ payload: checked, type: 'isDynamicSSHInjection' });
```

Comment:
Please add the type to the actions here https://github.com/kubevirt-ui/kubevirt-plugin/blob/main/src/views/catalog/CreateFromInstanceTypes/state/utils/types.ts#L35